docs/source/create_channel/create_channel.md

```diff
@@ -0,0 +1,383 @@
+
+You can download the `configtxgen` binaries by following the steps to [install the samples, binaries and Docker images](../install.html). `configtxgen` will be downloaded to the `bin` folder of your local clone of the `fabric-samples` repository along with other Fabric tools.
+
+For the purposes of this tutorial, we will want to operate from the `test-network` directory inside `fabric-samples`. Navigate to that directory using the following command:
+```
+cd fabric-samples/test-network
+```
+We will operate from the `test-network` directory for the remainder of the tutorial. Use the following command to add the configtxgen tool to your CLI path:
+```
+export PATH=${PWD}/../bin:${PWD}:$PATH
+```
+
+In order to use `configtxgen`, you need to the set the `FABRIC_CFG_PATH` environment variable to your local copy of the `configtx.yaml` file. For this tutorial, we will reference the `configtx.yaml` used to setup the Fabric test network in the `configtx` folder:
+```
+export FABRIC_CFG_PATH=${PWD}/configtx
+```
+
+You can check that you can are able to use the tool by printing the `configtxgen` help text:
+```
+configtxgen --help
+```
+
+## The configtx.yaml file
+
+The `configtx.yaml` file specifies the **channel configuration** that is contained inside the channel genesis block. The information that is required to build the channel configuration is specified in a readable and editable form in the `configtx.yaml` file. The `configtxgen` tool uses the channel profiles defined in the `configtx.yaml` file to create the channel configuration and write it to the [protobuf format](https://developers.google.com/protocol-buffers) that can be read by Fabric.
+
+You can find the `configtx.yaml` file that is used to deploy the test network in the `configtx` folder in the `test-network` directory. The file contains the following information that we will use to create our new channel:
+
+- **Organizations:** The organizations that can become members of your channel. Each organization has a reference to the cryptographic material that is used to build the [channel MSP](../membership/membership.html).
+- **Ordering service:** Which ordering nodes will form the ordering service of the network, and consensus method they will use to agree to a common order of transactions. The file also contains the organizations that will become the ordering service administrators.
+- **Channel policies** Different sections of the file work together to define the policies that will govern how organizations interact with the channel and which organizations need to approve channel updates. For the purposes of this tutorial, we will use the default policies used by Fabric.
```

Comment:
With the same caveat about the `-baseProfile` flag, all policies except for those in the Application section are ignored when generating a channel creation transaction.

Reply:
same comment as above